### PR TITLE
LocalBearTestHelper:Add settings to check_validity

### PR DESCRIPTION
**Pull request by refeed**
Add `settings` argument to `LocalBearTestHelper.check_validity`
and `LocalBearTestHelper.check_invalidity`.

Closes https://github.com/coala/coala/issues/4247

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: coalib/testing/LocalBearTestHelper.py [hunk 1]
```diff
@@ -79,14 +80,16 @@ def check_validity(self,
                                check_order=True,
                                force_linebreaks=force_linebreaks,
                                create_tempfile=create_tempfile,
-                               tempfile_kwargs=tempfile_kwargs
+                               tempfile_kwargs=tempfile_kwargs,
+                               settings=settings
```
Comment (jayvdb) [inline coalib/testing/LocalBearTestHelper.py]: add trailing comma

File: coalib/testing/LocalBearTestHelper.py [hunk 2]
```diff
@@ -79,14 +80,16 @@ def check_validity(self,
...
                                )
         else:
             return self.check_invalidity(local_bear, lines,
                                          filename=filename,
                                          force_linebreaks=force_linebreaks,
                                          create_tempfile=create_tempfile,
                                          tempfile_kwargs=tempfile_kwargs,
+                                         settings=settings
```
Comment (jayvdb) [inline coalib/testing/LocalBearTestHelper.py]: trailing comma

File: coalib/testing/LocalBearTestHelper.py [hunk 3]
```diff
@@ -120,7 +124,8 @@ def check_invalidity(self,
                                   filename=filename,
                                   force_linebreaks=force_linebreaks,
                                   create_tempfile=create_tempfile,
-                                  tempfile_kwargs=tempfile_kwargs)
+                                  tempfile_kwargs=tempfile_kwargs,
+                                  settings=settings)
```
Comment (jayvdb) [inline coalib/testing/LocalBearTestHelper.py]: move the closing `)` to the next line, and add a trailing comma on this line.

**Comment by jayvdb** [comment]
add needs rebase ;-)

**Comment by refeed** [comment]
updated

**Comment by jayvdb** [comment]
ack 6a7db15

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**